expat/lib/xmlparse.c

```diff
@@ -6827,3 +6838,24 @@ getElementType(XML_Parser parser,
   }
   return ret;
 }
+
+static XML_Char *
+copyString(const XML_Char *s,
+           const XML_Memory_Handling_Suite *memsuite)
+{
+    int charsRequired = 0;
+    XML_Char *result;
+
+    /* First determine how long the string is, including the terminator */
+    while (s[charsRequired] != 0) {
+      charsRequired++;
+    }
+    charsRequired++;
```

Comment:
Let's insert a blank line after (to communicate grouping) and maybe also move the note on the terminator into the `charsRequired++;` line, since `including the terminator` does not target the loop.